sprint0/sprint0/Enemies/Zol.cs

```diff
@@ -45,8 +46,20 @@ public Zol(Texture2D texture, Vector2 location, Color gelColor, Game1 game) : ba
 
         public override void Draw(SpriteBatch spriteBatch)
         {
-            if (damageTimer % 2 == 0)
-                spriteBatch.Draw(Texture, Location, colorMap[color][currentFrame / repeatedFrames], Color.White);
+            if (frameSpawn >= totalFramesSpawn * repeatedFramesSpawn)
+            {
+                if (damageTimer % 2 == 0)
+                {
+                    spriteBatch.Draw(Texture, Location, colorMap[color][currentFrame / repeatedFrames], Color.White);
+                }
+            }
+            else
+            {
+                if (frameSpawn < totalFramesSpawn * repeatedFramesSpawn)
+                {
+                    spriteBatch.Draw(game.Content.Load<Texture2D>("Images/Link"), Location, sourcesSpawn[frameSpawn / repeatedFramesSpawn], Color.White);
```

Comment:
we can probably just add this texture to the enemy factory or something. there are some factories w/multiple textures